server/services/courseService.ts

```diff
@@ -15,8 +15,13 @@ export default class CourseService {
     return courseClient.find(id)
   }
 
-  async getOfferingsByCourse(token: string, courseId: Course['id']): Promise<Array<CourseOffering>> {
+  async getOfferingsByCourse(token: string, id: Course['id']): Promise<Array<CourseOffering>> {
```

Comment:
This rename makes sense, as we're getting by the `course` so I think it's implied that `id` refers to Course here.